libs/template/presets/fulfillment/experience/pages/login-page.ts

```diff
@@ -25,16 +25,12 @@ export const fulfillmentLoginPage: ExperienceComponent = {
     {
       type: 'oryx-content-image',
       content: { data: { graphic: 'logo' } },
-      options: { rules: [{ height: '82px', align: 'start' }] },
+      options: { rules: [{ height: '82px', align: 'center' }] },
     },
     {
       type: 'oryx-content-text',
-      content: {
-        data: {
-          text: `<h3>${i18n('login.welcome')}</h3>`,
-        },
-      },
-      options: { rules: [{ padding: '0 0 20px' }] },
+      content: { data: { text: `<h3>${i18n('login.welcome')}</h3>` } },
```

Comment:
As I remember initially there was a text `login.welcome-please-log-in-to-start-picking` and now it's `Welcome` text only. Was it shortened by purpose?

Reply:
We like to keep all tokens short and logical, and have the translations/text inside our labs package. At least not have it in our code. We'll add the text to our labs package and likely have a labs deployment for FA as well.